04_week_optional/easy.js

```diff
@@ -0,0 +1,76 @@
+// Easy
```

Comment:
Nu știu de ce mi-a pus file-urile din tema 4 optională aici , le am deja în main...

Reply:
poate ca nu sunt in acelasi path... asta este